tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "allowSyntheticDefaultImports": true,
+    "target": "es6",
```

Comment:
We should change this depending which Node version(s) we're targeting. This is a good reference: https://stackoverflow.com/questions/59787574/typescript-tsconfig-settings-for-node-js-12

Reply:
That was a great reference thanks - going to go with
```
     "lib": ["es2019", "es2020.bigint", "es2020.string", "es2020.symbol.wellknown"],
    "module": "commonjs",
    "target": "es2019"
```
to target 12.0.0